src/runtime/pythonengine.cs

```diff
@@ -318,7 +310,16 @@ public static IntPtr InitExt()
         {
             try
             {
-                Initialize(setSysArgv: false, mode: ShutdownMode.Extension);
+                if (Runtime.IsInitialized)
+                {
+                    using var builtins = Runtime.PyImport_ImportModule("builtins");
+                    using var xx = Runtime.PyObject_GetAttrString(builtins.Borrow(), "RuntimeError");
+                    Exceptions.SetError(xx.Borrow(), ".NET Runtime is already initialized");
```

Comment:
1. The error should be something like `"Python.NET runtime is already initialized"`
2. We should probably use `PyEval_GetBuiltins` everywhere

Reply:
Changed error message, switched to `PyEval_GetBuiltins` here